spec/lib/tasks/deployment/migrate_prestation_choices_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+require 'support/after_party_helper'
+
+describe '20170504092142_migrate_prestation_choices' do
+  include_context 'after_party'
+
+  let!(:projet)     { create :projet }
+  let!(:prestation) { create :prestation }
+
+  it "charge l'environment Rails" do
+    subject.invoke
+
+    expect(subject.prerequisites).to include 'environment'
+  end
+
+  context "quand la table de jointure est vide" do
+    let!(:prestation_choice) { create :prestation_choice, projet: projet, prestation: prestation }
+
+    it "définie une prestation comme sélectionnée" do
+      subject.invoke
+
+      prestation_choice.reload
+      expect(prestation_choice.wished).to      be false
+      expect(prestation_choice.recommended).to be false
+      expect(prestation_choice.selected).to    be true
+    end
+  end
+
+  context "quand la table de jointure est déjà remplie" do
+    let!(:prestation_choice) { create :prestation_choice, :wished, projet: projet, prestation: prestation }
+
+    it "définie une prestation comme sélectionnée" do
+      subject.invoke
+
+      prestation_choice.reload
+      expect(prestation_choice.wished).to      be true
+      expect(prestation_choice.recommended).to be false
+      expect(prestation_choice.selected).to    be false
+    end
+  end
+
+  context "quand la table de jointure est erronée" do
+    let!(:prestation_choice_without_prestation) { create :prestation_choice, projet: projet, prestation: nil }
+    let!(:prestation_choice_without_projet)     { create :prestation_choice, projet: nil,    prestation: prestation }
```

Comment:
Oui, voilà… La différence de lisibilité ne serait pas flagrante, en même temps. Mais le mettre dans le `it`, ça permet aux gens qui ne connaissent pas l'usage un peu finaud de `let!` de comprendre le test quand même.

Je dirais qu'on laisse comme ça pour l'instant, et qu'on se repose la question la prochaine fois qu'on en aura besoin :)